### PR TITLE
Add the square property and fix the math to work for any given size

### PR DESCRIPTION
**Pull request by dilbertKocik**
This PR fixes the issue described [here](https://github.com/awesomejerry/react-native-qrcode-svg/issues/108).

### CODE REVIEW

File: src/transformMatrixIntoPath.js [hunk 1]
```diff
@@ -1,20 +1,21 @@
 export default (matrix, size) => {
   const cellSize = size / matrix.length
+  const adjustSize = size * 0.024
```
Comment (bryanaka) [inline src/transformMatrixIntoPath.js:3]: what is the multiple here stand for?

**Comment by bryanaka** [comment]
If it works :shipit:

File: src/transformMatrixIntoPath.js (hunk 1 above)
Comment (dilbertKocik) [reply src/transformMatrixIntoPath.js:3]: It's a magic ratio that you need to multiply by any given size and add or subtract depending on which section is being drawn. I can't explain how it got broken but this fixes the issue for sizes greater than and smaller than the default size. It's magic.

Comment (dilbertKocik) [reply src/transformMatrixIntoPath.js:3]: The way I found it is I kept messing with size adjustment manually until I found the right answer for what works for us. Which happens to be 6. Then I realized that number must be related to the size and just guessed that size * someMagicNumber = 6. Solve for someMagic number and you get the simple equation above. Like I said, it works for all sizes I tested.

**Pull request merged**